assets/out

```diff
@@ -100,10 +102,19 @@ elif [ -n "$build" ]; then
     docker_pull "${repository}:${cache_tag}"
   fi
 
+  expanded_build_args=""
+  if [ -n "$build_args" ]; then
+    expanded_build_args=$(echo $build_args|jq -r 'with_entries(.key |= "--build-arg " + . )|with_entries(.key = .key + "=" +.value)|keys|join(" ")')
```

Comment:
Indeed, it could be nice to rewrite it in Go, but I'm not ready to do it yet. Maybe later :smile: